packages/with-yup/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@remix-validated-form/with-yup",
+  "version": "1.0.0",
+  "browser": "./browser/index.js",
+  "main": "./build/index.js",
+  "scripts": {
+    "dev": "tsc --module ESNext --outDir ./browser --watch",
+    "build": "npm run build:browser && npm run build:main",
+    "build:browser": "tsc --module ESNext --outDir ./browser",
+    "build:main": "tsc --module CommonJS --outDir ./build",
+    "prepublishOnly": "cp ../../README.md ./README.md && npm run build",
+    "postpublish": "rm ./README.md"
+  },
+  "peerDependencies": {
+    "remix-validated-form": "^3.x",
```

Comment:
Does this fail to match on the current `beta` tag (https://www.npmjs.com/package/remix-validated-form/v/3.0.0-beta.1) because the `-beta...` suffix doesn't conform to strict SemVer?